it/connectors/ArrivalConnectorSpec.scala

```diff
@@ -205,7 +205,7 @@ class ArrivalConnectorSpec
       val connector = app.injector.instanceOf[ArrivalConnector]
       val arrival = Arrival(
         ArrivalId(1),
-        routes.ArrivalMovementController.getArrival(ArrivalId(1)).urlWithContext,
+        routing.routes.ArrivalsRouter.getArrival(ArrivalId(1).toString).urlWithContext,
```

Comment:
we can pass directly string here "1" instead of ArrivalId(1).toString? because in ArrivalRouter we are now passing string directly